frontend/src/components/LoginPage.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import TUMLogo from "../assets/tum-logo.svg"; // 确保路径正确
+
+const LoginPage = () => {
+    const [email, setEmail] = useState("");
+    const [error, setError] = useState("");
+
+    const handleSignIn = () => {
+        const emailRegex = /^[^\\s@]+@[^\\s@]+\\.[^\\s@]+$/;
+        if (!emailRegex.test(email)) {
+            setError("Invalid Account, please try again!");
+        } else {
+            setError("");
+            console.log("Sign-in with email:", email);
+        }
+    };
+
+    return (
+        <div
+            className="relative flex justify-center items-center min-h-screen"
+            style={{
+                backgroundColor: "#f5f5f5", // 设置页面背景色
+            }}
+        >
+            {/* 左上角 TUM Logo */}
```

Comment:
NIT: The comment can be ommited

https://www.alibabacloud.com/blog/the-art-of-code-comments-does-good-code-need-comments_600184

Reply:
If you think the comment can help, you may write in english

Reply:
good advice